driver/Dockerfile.hail

```
@@ -3,8 +3,8 @@ FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver-base:1.2
 
 ARG HAIL_SHA
 
-ENV HAIL_QUERY_BACKEND service \
-    TZ=Australia
+ENV HAIL_QUERY_BACKEND service
+ENV TZ="Australia"
```

I guess both ways work, but I'd use the same format for consistency:
```suggestion
ENV TZ Australia
```

---

(also following the format of these statements https://github.com/populationgenomics/analysis-runner/blob/3c4f1e4ac95ec2f33503d81677bec1144553746a/driver/Dockerfile.r)